### ISSUE TITLE
No Bootable Medium Found: File not found

### ISSUE THREAD

**Issue by derpyParticle311** [MightyPirates/OpenComputers#420]
i am using MC1.7.10 with OC1.3.1.28. i have disabled power consumption because when i have a advanced universal cable connected with power, it still says 'Not enough Energy' in chat. That is weird but i just stuck with no energy consumption. The real problem is that when i have a T1CPU, T1MemoryCard, T1HardDrive, and T1GraphicsCard in my T1ComputerCase and T1Monitor. I have a keyboard just incase it begins working. When i press on (the modules are installed at this time) It will quickly appear as 'Turn Off' but then immediately turns back to 'Turn On'. The monitor gives this error message: 'Unrecoverable Error no bootable medium found: file not found
I would appreciate some help because i really want to get to know this mod! it looks like its got a bright blocky future ahead. :D good day.

**Comment by Techokami** [comment]
As of OpenComputers 1.3.0, you need to craft an OS install floppy. (ROM is dead!)
To do that, just craft a blank floppy with a book. OR, you might find it in a loot chest in dungeons and what not. Then just craft and place a Disk Drive next to your computer case, and insert the disk. You computer will now boot! :D then you run the installer program (type install and press enter) to install the OS to your hard drive.

**Comment by fnuecke** [comment]
Exactly what Techokami said. With the remark that installing the OS on a disk is optional, so long as you don't intend to write files to the root directory or change OS files (and don't mind keeping the disk drive around).

As for the power issues, it seems Mekanisms power system implementations aren't quite fully ported, yet (it doesn't connect to IC2 for me, and it crashes when I place a BC wooden kinesis pipe next to an energy cell). That's not fully representative, though, because I only tested this in dev-env. I did however add direct support for Mekanism's own power system just now, so expect that in 1.3.2 of OC. For now use IC2 or BC directly to power OC.
